pkg/apis/operators/v1alpha1/servicebinding_types.go

```diff
@@ -35,6 +35,15 @@ type ServiceBindingSpec struct {
 	// different subresources owned by backing operator CR.
 	// +optional
 	DetectBindingResources *bool `json:"detectBindingResources,omitempty"`
+
+	// BindAsFiles flag enable binding values as files.
+	// If `SERVICE_BINDING_ROOT` env var is present, mountPath is ignored.
```

Comment:
mountPath is ignored? We might need to mention 
SERVICE_BINDING_ROOT` env var is present, mountPath is present. That time what takes precedence. 
basically there are these:
SBR | MountPath
T      |  F
T      |  T
F      |  F
F      |  T

I see 3 conditions being mentioned

Reply:
when both are true, we need that also

Reply:
The statement "... mountPath is ignored" handles both two cases of `mountPath` value.

Reply:
Ok, I will move the doc up.
The `optional` mark makes it possible to provide YAML without these fields. The `mountPath` also marked as `optional`.

Reply:
Done. 172198aa20026266b5b017693826e754be0f1567